src/error.php

```diff
@@ -11,25 +11,23 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0">
 
-    <link rel="preload" href="<?=$config['upload_access_path']?>js/setTheme.js" as="script">
+    <link rel="preload" href="js/setTheme.js" as="script">
 
-    <link rel="stylesheet" href="<?=$config['upload_access_path']?>css/toggle-bootstrap.min.css">
-    <link rel="stylesheet" href="<?=$config['upload_access_path']?>css/toggle-bootstrap-dark-overlay.min.css" onload="this.disabled = true; window.__darkCssLoaded = true; if (window.__updateTheme) { window.__updateTheme(); }">
+    <link rel="stylesheet" href="css/toggle-bootstrap.min.css">
+    <link rel="stylesheet" href="css/toggle-bootstrap-dark-overlay.min.css" onload="this.disabled = true; window.__darkCssLoaded = true; if (window.__updateTheme) { window.__updateTheme(); }">
 
-    <link rel="stylesheet" href="<?=$config['upload_access_path']?>css/main.css">
+    <link rel="stylesheet" href="css/main.css">
     <title>Error - <?php echo $config['page_title']; ?></title>
     
-    <link rel="apple-touch-icon" sizes="180x180" href="<?=$config['upload_access_path']?>icons/apple-touch-icon.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="<?=$config['upload_access_path']?>icons/favicon-32x32.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="<?=$config['upload_access_path']?>icons/favicon-16x16.png">
-    <link rel="manifest" href="<?=$config['upload_access_path']?>manifest.webmanifest">
-    <link rel="mask-icon" href="<?=$config['upload_access_path']?>icons/safari-pinned-tab.svg" color="#5c5cbc">
-    <link rel="shortcut icon" href="<?=$config['upload_access_path']?>favicon.ico">
+    <link rel="apple-touch-icon" sizes="180x180" href="icons/apple-touch-icon.png">
```

Comment:
You've kind of merged your two branches together, once you've merged in #106 I'll come back and review this :)

Reply:
Haha you're right, I must have not branched off from master.